src/components/bookRender.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { api } from "~/utils/api";
+import { BellIcon, CheckIcon } from "@radix-ui/react-icons";
+
+import { cn } from "~/lib/utils";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+type CardProps = React.ComponentProps<typeof Card>;
+
+const BookRender = (): JSX.Element => {
+  return ShadCard({});
+};
+
+export function ShadCard({ className, ...props }: CardProps) {
```

Comment:
not sure if we want this to be ShadCard. Open to it but i don't understand whats going on